src/components/Identity/createIdSyncs.js

```diff
@@ -37,7 +37,7 @@ const setControlObject = (controlObject, cookieJar) => {
 };
 
 const createProcessor = (config, logger, cookieJar) => destinations => {
-  if (!config.idSyncsEnabled) {
+  if (!config.idSyncEnabled) {
     return Promise.reject(new Error("ID syncs not enabled"));
```

Comment:
You might want to change this error message to "ID sync not enabled" or "ID synchronization not enabled".

Reply:
Thanks, I'll use the second one.